codec-http2/src/main/java/io/netty/handler/codec/http2/Http2GoAwayFrame.java

```diff
@@ -46,6 +49,11 @@
     Http2GoAwayFrame setExtraStreamIds(int extraStreamIds);
 
     /**
+     * Returns the last stream identifier if set, or {@code -1} else.
```

Comment:
clarify the limitation enforced by the [Http2FrameCodec](https://github.com/netty/netty/pull/5324/files#diff-004b9d3b44cadc4c6987434ba0e599beR174) (that this must be `-1` when sending)?